src/App.js

```diff
@@ -1,12 +1,24 @@
 import React from 'react';
 import {BrowserRouter as Router, Route, Switch, Redirect} from "react-router-dom";
-import {Login} from "./Login"
+
 import {Dashboard} from "./Dashboard";
-import {Register} from "./Register";
+import Register from "./Register";
+import Login from "./Login"
+import {createAuthProvider} from "react-token-auth";
+
+// export const [useAuth, authFetch, login, logout] =
+// createAuthProvider({
+//     accessTokenKey: 'accessToken',
+// });
+export const [useAuth, authFetch, login, logout] =
+createAuthProvider({
+    accessTokenKey: 'accessToken',
+    refreshTokenKey: 'refreshToken'
```

Comment:
Actually, I am not sure it will be enough to update token, because `react-token-auth` knows nothing about the services you have on the server.

On the our server there is a router `/auth/update-token` (should be checked in the swagger), so you can use this router in the next way:

```
onUpdateToken: (token) => fetch('/auth/update-token', {
            method: 'POST',
            body: token.refreshToken
        })
        .then(r => r.json())
```